### PR TITLE
Database

### PR DESCRIPTION
**Pull request by Leoli0320**

### CODE REVIEW

File: db/migration/001_tweets_create.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+require 'sequel'
+
+Sequel.migration do
+  change do
+    create_table(:tweets) do
+      primary_key :id
+      String :tweet_id
+      String :create_time
```
Comment (cyhsutw) [inline db/migration/001_tweets_create.rb]: Can we change this to `datetime` data type? 
Also, I would prefer the name of this field to be `created_at`.

**Review by cyhsutw — CHANGES_REQUESTED** [review]
@Leoli0320 thanks for this pull request.

I pull this branch and run `rake db:migrate` locally, and it isn't working.

I think you'll need to move `db/migration/001_tweets_create.rb` to `db/migrations/001_tweets_create.rb` (you're missing an `s`).

Another minor nit, could you run `bundle install` and commit the new `Gemfile.lock`?

**Comment by Leoli0320** [comment]
@cyhsutw  thanks for point out the problems! I'm very appreciated.
I correct the error that you found. 
Could you help me check it again?
Thanks!

**Comment by cyhsutw** [comment]
Closing this as we are not going to maintain this API.

**Pull request closed without merging**